doc/api/tls.md

```diff
@@ -1406,7 +1406,9 @@ to `true`, other APIs that create secure contexts leave it unset.
 from `process.argv` as the default value of the `sessionIdContext` option, other
 APIs that create secure contexts have no default value.
 
-The `tls.createSecureContext()` method creates a credentials object.
+The `tls.createSecureContext()` method creates a `SecureContext` object. The
+object has no public methods, but is accepted as an argument to several `tls`
```

Comment:
Total micro-nit that you can ignore if you disagree, but I think the two sentences are clearer if you divide them up this way instead:

```markdown
The `tls.createSecureContext()` method creates a `SecureContext` object with
no public methods. It is usable as an argument to several `tls` APIs such as
[`tls.createServer()`][] and [`server.addContext()`][].
```

Reply:
I see your point, how about ba029f50de?